cdc/sink/mq.go

```diff
@@ -185,23 +231,16 @@ flushLoop:
 	for {
 		select {
 		case <-ctx.Done():
-			return 0, ctx.Err()
+			return ctx.Err()
 		case <-k.resolvedReceiver.C:
```

Comment:
How often the tick `C` is?

Reply:
Every time a `mqSikn.worker` flushes the producer, we will receive a `C` here.